src/pages/AdminPage/Admin.tsx

```diff
@@ -67,7 +68,7 @@ function Admin() {
   };
 
   const handleReject=async (job: Job)=>{
-    await axios.put(REACT_BACKEND_ROUTE+`/v1/admin/${job._id}`,{status:'Rejected'})
+    await axios.put(REACT_BACKEND_ROUTE+`v1/admin/${job._id}`,{status:'Rejected'})
```

Comment:
we can use REACT_APP_BACKEND_URL every where
wrap the api call as `${REACT_BACKEND_ROUTE}/v1/admin/'